packages/@react-types/shared/src/dnd.d.ts

```diff
@@ -119,11 +119,20 @@ export interface DragTypes {
 }
 
 export interface DroppableCollectionProps {
+  /**
+   * A function returning the drop operation to be performed when items matching the given types are dropped
+   * on the drop target.
+   */
   getDropOperation?: (target: DropTarget, types: DragTypes, allowedOperations: DropOperation[]) => DropOperation,
+  /** Handler that is called when a valid drag element enters the drop target. */
   onDropEnter?: (e: DroppableCollectionEnterEvent) => void,
+  /** Handler that is called when a valid drag element is moved within the drop target. */
   onDropMove?: (e: DroppableCollectionMoveEvent) => void,
+  /** Handler that is called after a valid drag element is held over the drop target for 800 ms. */
```

Comment:
Probably should specify the exact time here 😄

Reply:
Should or shouldn't?

Reply:
sorry missed this. I meant should not. But we can change this later.

Reply:
sounds good, I'll open a follow up PR